Use the Google Form's spreadsheet as the data source.

isaprykin opened the pull request:
The data is now displayed from the same Google Sheet as the one where the Google Form writes.
The format is also changed since this spreadsheet has more columns.  The column headers are updated to match the spreadsheet.

isaprykin committed:
Use the Google Form's spreadsheet as the data source.

The data is now displayed from the same Google Sheet as the one where the Google Form writes.
The format is also changed since this spreadsheet has more columns.  The column headers are updated to match the spreadsheet.

sunnyeyez123 commented on index.md:
```diff
@@ -34,9 +34,10 @@ Thanks is a page where Igor and Jasmine can log and view messages of graditude t
+      <th>Timestamp</th>
+      <th>Who do you want to thank?</th>
+      <th>What are you thankful for?</th>
+      <th>Who are you?</th>
```
Can you change this to Thanker

sunnyeyez123 commented on index.md:
```diff
@@ -34,9 +34,10 @@ Thanks is a page where Igor and Jasmine can log and view messages of graditude t
-      <th>Name</th>
-      <th>Date</th>
-      <th>Thanks</th>
+      <th>Timestamp</th>
```
Can you change this to "Date"

sunnyeyez123 commented on index.md:
```diff
@@ -34,9 +34,10 @@ Thanks is a page where Igor and Jasmine can log and view messages of graditude t
-      <th>Date</th>
-      <th>Thanks</th>
+      <th>Timestamp</th>
+      <th>Who do you want to thank?</th>
```
This should be "Thankee"

isaprykin committed:
Remove the HTML table border.

It's done by setting cellspacing="0" cellpadding="0" on the HTML table.
Alternatively, we could have done in CSS by setting
"table {border: none;}".

isaprykin committed:
Update the column headers.

Column headers are updated in response to the code review comments.

isaprykin reviewed (commented):
Thanks for the review. Please take another look.
